### ISSUE TITLE
[BUG] Files with .pm extension no longer recognized as Perl

### ISSUE THREAD

**Issue by SamInPgh** [notepad-plus-plus/notepad-plus-plus#15365]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of the Issue

Until the most recent version (v8.6.8), Notepad++ would automatically recognize my .pm files as Perl when opening them from Windows File Explorer and format them accordingly. Now I must manually select 'Perl' from the Language menu after opening them. However, they are formatted correctly when I open them from the Notepad++ File menu.

### Steps To Reproduce

1.  Select a .pm Perl module from File Explorer and open with Notepad++
2.  
3.  

### Current Behavior

The file is not recognized as a Perl module and no language formatting is applied.

### Expected Behavior

Perl language formatting should be automatically applied.

### Debug Information

```shell
Notepad++ v8.6.8   (64-bit)
Build time : Jun  4 2024 - 00:30:00
Path : D:\Program Files\Notepad++\notepad++.exe
Command Line : "D:\Github\slimserver\Slim\Player\ReplayGain.pm"
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
Periodic Backup : ON
OS Name : Windows 11 Home (64-bit)
OS Version : 23H2
OS Build : 22631.3737
Current ANSI codepage : 1252
Plugins : 
    NppExport (0.4)
    mimeTools (3.1)
    NppConverter (4.6)
```

**Comment by CennoxX** [comment]
I can't reproduce this with v8.6.8 and Windows 10. 
Could you please compare your "C:\Program Files\Notepad++\langs.model.xml" and https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/src/langs.model.xml (In particular this line `<Language name="perl" ext="pl pm plx t" commentLine="#">`)?

**Comment by SamInPgh** [comment]
Thanks for the response. I figured out what was going on. As part of upgrading to v8.6.8, I moved the install location from my C drive to my D drive. Apparently, Windows didn't get the message and was still associating files of type '.pm' to the version of Notepad++ that remained on my C drive, which resulted in the mixup. Bottom line: not a Notepad++ problem. Deleting the old version on my C drive. Thanks for your help.
